addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -80,16 +80,31 @@ body.editor_has_snippets .o_field_mass_mailing_html div.d-flex:has(iframe) {
         }
         &.o_kanban_ungrouped {
             padding: 0;
-            .o_mailing_list_kanban_grouped {
-                display: none !important;
-            }
             .o_kanban_record {
                 width: 100%;
                 margin: 0;
             }
         }
-        &.o_kanban_grouped .o_mailing_list_kanban_ungrouped {
-            display: none !important;
+        &.o_kanban_grouped {
+            .flex-row > div {
+                flex: 1 1 100% !important;
+                max-width: 100% !important;
+            }
+            .flex-row > div.col-6 {
+                flex: 1 1 50% !important;
+                max-width: 50% !important;
+            }
+            .flex-row > div.d-none {
+                display: none !important;
+            }
+            .o_mailing_list_kanban_stats > a {
```

Comment:
Looks like a lot of these rules are copied from ".o_mailing_list_kanban_stats > a { ... @include media-breakpoint-down(sm)".

there is probably a way to have common rules for both the grouped use case and the smaller resolution case?